### PR TITLE
add fixes to sdk-feature-store.ipynb (formatting, link edits)

### PR DESCRIPTION
**Pull request by nayaknishant**

### CODE REVIEW

File: notebooks/official/feature_store/sdk-feature-store.ipynb [hunk 1]
```diff
@@ -40,7 +40,7 @@
         "      <img src=\"https://cloud.google.com/ml-engine/images/github-logo-32px.png\" alt=\"GitHub logo\">\n",
```
Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:40]: <p>There might be something wrong with the Vertex AI workbench link.</p><p> </p><p>Add H1 heading in same cell, but above links</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/913/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:40]: <p>Split into 4 separate cells.</p><p>Move Dataset after objective</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/913/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:40]: <p>use template version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/913/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:40]: <p>use newer UUID template version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/913/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:40]: <p>remove 'Skip this step'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/913/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:40]: <p>Add a heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/913/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:40]: <p>Add a heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/913/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:40]: <p>Add text cell to explain what next 2 code cells are for.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/913/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:40]: <p>Add heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/913/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:40]: <p>GCS (Google Cloud Storage) -&gt; Cloud Storage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/913/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:40]: <p>Python SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/913/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:40]: <p>Add heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/913/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

**Comment by ivanmkc** [comment]
I think Nishant just wanted to fix the one broken link. If there are further issues with the notebook, we can fix it in a separate PR. You can log the issues in Github Issues.

File: notebooks/official/feature_store/sdk-feature-store.ipynb (hunk 1 above)
Comment (nayaknishant) [reply notebooks/official/feature_store/sdk-feature-store.ipynb:42]: Fixed

Comment (nayaknishant) [reply notebooks/official/feature_store/sdk-feature-store.ipynb:42]: Fixed

Comment (nayaknishant) [reply notebooks/official/feature_store/sdk-feature-store.ipynb:42]: Fixed

Comment (nayaknishant) [reply notebooks/official/feature_store/sdk-feature-store.ipynb:42]: Fixed

**Comment by nayaknishant** [comment]
Thanks @ivanmkc. It seems like majority of the comments were addressed with https://github.com/GoogleCloudPlatform/vertex-ai-samples/pull/790. I added the remaining fixes and also fixed the URL links. As for the comments about the headings, I don't think the extra headings are needed because many of the code blocks are referencing headings from before or are introducing simple methods like `get` that don't need a heading and take away from the notebook workflow. Please take a look when you get the chance @andrewferlitsch.

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**